rentals.json

```diff
@@ -0,0 +1 @@
+[{"person_id":6664,"book_title":"what","rental_date":"2005-03-23"}]
```

Comment:
- [ ] Great job creating this JSON file to store rental data, but according to the project requirements, you should create JSON files for all the entities you want to save and make the save data as per the requirement. Currently, nothing is saved, I say so because when I choose option 7 to quit the app, and then I run it again, I don't see the saved data. Kindly work on this issue and submit it again. __Remember to have equal git commits__

![preserve-data](https://github.com/Rishi-Mishra0704/ruby-class/assets/53124548/17036bfc-b9b3-4b68-b1e6-99b4baf39860)

![preserve](https://github.com/Rishi-Mishra0704/ruby-class/assets/53124548/a2926448-d67c-4f59-9d60-e453e1caa245)